.pylintrc-mandatory.jinja

```diff
@@ -38,8 +38,6 @@ enable=anomalous-backslash-in-string,
     method-inverse,
     method-required-super,
     method-search,
-    missing-import-error,
-    missing-manifest-dependency,
```

Comment:
Maybe we could add `missing-manifest-dependency` to the optional linters?

Reply:
Yep, I was considering that too. Done.